README.md

```diff
@@ -1,2 +1,237 @@
+- `stacks:`
+    - `name:` The name of a repository to build, which might contain one or more application stacks.
+    - `repos:` An array of URLs that point to one or more stack index files that you want to include in this repository
+        - `url:` `exclude:` (Optional) An array of stack names that you want to exclude from the repository. If no filtering is required, leave blank.
+        - `url:` `include:` (Optional) An array of stack names that you want to include from the repository. If no filtering is required, leave blank.
+- `image-org:` (Optional) The name of the organization within the image registry where you want to store the docker images for included stacks. See **Configuring public or private repositories**.
+- `image-registry:` (Optional) The URL of the image registry where you want to store the stack docker images. See **Configuring public or private repositories**.
+- `nginx-image-name:` (Optional) The name assigned to the generated NGINX image. The default is `repo-index`. See **Configuring public or private repositories**.
+
+NOTE: The `exclude:` and `include:` fields are mutually exclusive. Use only one or the other for a specific repository to avoid build errors.
+
+**Configuring public or private repositories**
+
+If your stacks and repositories are publicly available, leave the `image-org` and `image-registry` fields blank.
+
+If your stacks and repositories are hosted in a private environment that your deployment environment and tools cannot access, such as GitHub Enterprise, the build can create an NGINX image. This image can serve the assets you need from within your deployment environment without the need for authentication. Use the `image-org` and `image-registry` fields to define the organization and URL where you store your registry images. You can optionally configure a name for your NGINX image, which you must push to the registry that your deployment environment can access. When deployed, the NGINX image serves the repository index files that you create as part of the build.
+
+### Configuring your stack hub
+
+In this guide, you will change the example configuration file and use it build a stack hub definition. Follow these steps:
+
+1. Edit the `example_config/example_repo_config.yaml` file to include the following values for your stack hub:
+
+```
+# File: example_repo_config.yaml
+# My organization stack hub
+name: My.org Stack Hub
+description: Test configuration to build 2 index files for the default and incubator repos.
+version: 0.1.0
+stacks:
+  - name: default
+    repos:
+      - url: https://github.com/appsody/stacks/releases/latest/download/incubator-index.yaml
+        exclude:
+          - kitura
+          - node-red
+          - python-flask
+          - starter
+          - swift
+          - java-microprofile
+          - nodejs
+      - url: https://github.com/kabanero-io/collections/releases/download/0.6.0-rc.1/kabanero-index.yaml
```

Comment:
I would probably change this url to be https://github.com/kabanero-io/collections/releases/download/0.6.0/kabanero-index.yaml rather than the rc.1 url